[2023-07-11T15:06:45.881+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: AAAA.create_drivers_table scheduled__2023-07-11T00:00:00+00:00 [queued]>
[2023-07-11T15:06:45.892+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: AAAA.create_drivers_table scheduled__2023-07-11T00:00:00+00:00 [queued]>
[2023-07-11T15:06:45.893+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-07-11T15:06:45.913+0000] {taskinstance.py:1327} INFO - Executing <Task(PostgresOperator): create_drivers_table> on 2023-07-11 00:00:00+00:00
[2023-07-11T15:06:45.919+0000] {standard_task_runner.py:57} INFO - Started process 8226 to run task
[2023-07-11T15:06:45.924+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'AAAA', 'create_drivers_table', 'scheduled__2023-07-11T00:00:00+00:00', '--job-id', '227', '--raw', '--subdir', 'DAGS_FOLDER/import_data2.py', '--cfg-path', '/tmp/tmp8y2jldlz']
[2023-07-11T15:06:45.925+0000] {standard_task_runner.py:85} INFO - Job 227: Subtask create_drivers_table
[2023-07-11T15:06:45.990+0000] {task_command.py:410} INFO - Running <TaskInstance: AAAA.create_drivers_table scheduled__2023-07-11T00:00:00+00:00 [running]> on host 1d6232dcfa4d
[2023-07-11T15:06:46.097+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='AAAA' AIRFLOW_CTX_TASK_ID='create_drivers_table' AIRFLOW_CTX_EXECUTION_DATE='2023-07-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-07-11T00:00:00+00:00'
[2023-07-11T15:06:46.101+0000] {sql.py:265} INFO - Executing: DROP TABLE IF EXISTS drivers_data;
CREATE TABLE drivers_data (
    "school_year" TEXT,
    "vendor_name" TEXT,
    "type_of_service" TEXT,
    "active_employees" TEXT,
    "job_type" TEXT,
);
[2023-07-11T15:06:46.111+0000] {base.py:73} INFO - Using connection ID 'tutorial_pg_conn' for task execution.
[2023-07-11T15:06:46.192+0000] {base.py:73} INFO - Using connection ID 'tutorial_pg_conn' for task execution.
[2023-07-11T15:06:46.198+0000] {sql.py:375} INFO - Running statement: DROP TABLE IF EXISTS drivers_data;
CREATE TABLE drivers_data (
    "school_year" TEXT,
    "vendor_name" TEXT,
    "type_of_service" TEXT,
    "active_employees" TEXT,
    "job_type" TEXT,
);, parameters: None
[2023-07-11T15:06:46.200+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 277, in execute
    **extra_kwargs,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 349, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 380, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.SyntaxError: syntax error at or near ")"
LINE 8: );
        ^

[2023-07-11T15:06:46.215+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=AAAA, task_id=create_drivers_table, execution_date=20230711T000000, start_date=20230711T150645, end_date=20230711T150646
[2023-07-11T15:06:46.236+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 227 for task create_drivers_table (syntax error at or near ")"
LINE 8: );
        ^
; 8226)
[2023-07-11T15:06:46.256+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-07-11T15:06:46.297+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
